Add myself (@cstyan) as 2.16 release shepherd.

cstyan opened the pull request:
Note that I also changed the date of the release from the 29th to the 27th to leave the whole week open for release related work instead of just three-ish days.

Signed-off-by: Callum Styan <callumstyan@gmail.com>

roidelapluie commented:
AFAIK the date is the date of pre-release and there is nothing that says that the release must be cut 3 days after the pre-release (it just says run the benchmarks 3 days). We could cut the release one week after the pre-release.

roidelapluie commented:
(3 days of pre-release is quite short to let testers install and test the pre-release.)

cstyan commented:
I think the dates all being Wednesdays was just to match exactly to a 6 week cadence? Though the first release by @gouthamve was on a Thursday apparently. Anyways, I changed it back to the original date and added the next release to the list.
